packages/icons/manifest.json

```diff
@@ -986,6 +986,11 @@
       "category": "Finance",
       "size": "24"
     },
+    {
+      "name": "interest_of_balance",
```

Comment:
What is 'interest of balance' supposed to be? The[Google Search for it](https://www.google.com/search?q=%22interest+of+balance%22) does not yield anything plausible

Reply:
oops... Thank you for pointing the typo 😅